modules/sdk-coin-dot/src/lib/transaction.ts

```diff
@@ -18,18 +19,34 @@ import { KeyPair } from './keyPair';
 import {
   AddAnonymousProxyArgs,
   AddProxyArgs,
+  AddProxyBatchCallArgs,
   BatchArgs,
+  BatchCallObject,
   ClaimArgs,
   DecodedTx,
   HexString,
-  StakeArgs,
+  MethodNames,
+  SectionNames,
   StakeArgsPayeeRaw,
+  StakeBatchCallArgs,
   TransactionExplanation,
   TxData,
   UnstakeArgs,
   WithdrawUnstakedArgs,
 } from './iface';
+import { getAddress, getDelegateAddress } from './iface_utils';
 import utils from './utils';
+import BigNumber from 'bignumber.js';
+import { Vec } from '@polkadot/types';
+import { PalletConstantMetadataV14 } from '@polkadot/types/interfaces';
+
+const FAKE_SIGNATURE = `0x${Buffer.from(new Uint8Array(256).fill(1)).toString('hex')}`;
```

Comment:
nit : this doesnt have to be global variable?

Reply:
Fixed.